config/migrations/20220511133621-add-publication-report-types.ttl

```diff
@@ -0,0 +1,54 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix pub: <http://mu.semte.ch/vocabularies/ext/publicatie/> .
+
+@prefix PublicationReportTypeScheme: <http://themis.vlaanderen.be/id/concept-scheme/b75a85c3-a599-4d4f-a2a2-dea583f2fd86> .
+@prefix publicationReportType: <http://themis.vlaanderen.be/id/concept/publicatierapporttype/> .
```

Comment:
Be mindful of casing in URIs. Technically, they're case-sensitive, and inconsistencies might lead to issues depending on implementation. So would make this `PublicatieRapportType` as well.

(explanation on why URIs are case-sensitive: https://webmasters.stackexchange.com/questions/90339/why-are-urls-case-sensitive , if you're interested)